core/lib/presentation/views/file_viewer/pdf_viewer.dart

```diff
@@ -0,0 +1,66 @@
+
+import 'dart:math' as math;
+import 'dart:typed_data';
+
+import 'package:core/presentation/views/file_viewer/top_bar_pdf_viewer.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:syncfusion_flutter_pdfviewer/pdfviewer.dart';
+
+class PDFViewer extends StatelessWidget {
+  static const double _maxWidthSizeDefault = 750;
+
+  final Uint8List bytes;
+  final String? fileName;
+  final String? downloadTitle;
+  final String? printTitle;
+
+  const PDFViewer({
+    super.key,
+    required this.bytes,
+    this.fileName,
+    this.downloadTitle,
+    this.printTitle,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      type: MaterialType.transparency,
+      child: InkWell(
+        onTap: Navigator.maybeOf(context)?.pop,
+        child: Container(
+          color: Colors.transparent,
+          alignment: Alignment.center,
+          child: Stack(
```

Comment:
```suggestion
        child: Center(
          child: Stack(
```

Reply:
`Center` is more lightweight than `Container`

Reply:
Good detection. Done